### DISCUSSION TITLE
SN utils not working for instances with SSO login

### DISCUSSION THREAD

**Discussion by sKravciv** [arnoudkooi/SN-Utils#411] [Q&A] [answered]
Hi, is there something I can do to make the SN utils work on instances where is SSO used for authentication. Slash commands are not working and the plugin keeps asking for login informations. Thank You

**Comment by arnoudkooi** [comment, marked as answer]
Once logged via sso this should work. Are you on Firefox with the containers plugin?

**Reply by sKravciv** [reply]
yes

**Reply by arnoudkooi** [reply]
Then that is probably the issue, not the SSO.
I have looked at this but have not found a resolution so far.
The only workaround is having a tab open where you are logged in, in the default container.

**Reply by sKravciv** [reply]
OK thanks
